enemies/bosses/scenarios.json

```diff
@@ -70,6 +70,81 @@
       "incomingDamage": [],
       "note": "Botwoon 2 can not hit Samus while she is in the bottom left corner of the room."
     },
+    {
+      "id": 3,
+      "name": "Reverse Botwoon",
+      "boss": "Botwoon 1",
```

Comment:
I think this might only be referring to phase 1 of Botwoon. Phase 2 should be added as well without a spit attack.

Reply:
Without the acid attack Samus cannot be damaged anymore.  For the beam fight it is a free kill.

Ammo fights need to account for boss health though.  I want 6 PBs.  A super fight would be ~13.
It just works out that this dodge rate and this boss health takes the right PB count.  The super fight would have a Bot2 phase I think.